app/components/Footer/index.jsx

```diff
@@ -9,37 +9,22 @@ export default class Footer extends Component {
     return <footer id="Footer">
       <div className="footer-content">
         <div className="footer-links">
-          <ul>
-            <h4>Explore</h4>
-            <li><Link to="/search/">Reports</Link></li>
-            <li><Link to="/visualize">Viz Builder</Link></li>
-            <li><Link to="/map/">Maps</Link></li>
-            <li><Link to="/cart/">Data Cart</Link></li>
-          </ul>
-          <ul>
-            <h4>Sources</h4>
-            <li><Link to="/about/datasets/">Data Sources</Link></li>
-            <li><Link to="/about/api/">API</Link></li>
-            <li><Link to="/about/classifications/">Classifications</Link></li>
-            <li><a href="mailto:hello@datausa.io">Contact Us</a></li>
-          </ul>
-          <ul>
-            <h4>About</h4>
-            <li><Link to="/about/">Background</Link></li>
-            <li><Link to="/about/press/">In the Press</Link></li>
-            <li><Link to="/about/team/">Team</Link></li>
-            <li><Link to="/about/glossary/">Glossary</Link></li>
-            <li><Link to="/about/usage/">Terms of Use</Link></li>
-          </ul>
+          <img src="/images/logo_sm.png" alt="Data USA" />
+          <p>Explore, map, compare, and download U.S. data</p>
+          <div className="footer-column">
+            <div><h4><Link to="/">Home</Link></h4></div>
+            <div><h4><Link to="/search/">Reports</Link></h4></div>
+            <div><h4><Link to="/visualize">VizBuilder</Link></h4></div>
+            <div><h4><Link to="/map/">Maps</Link></h4></div>
+            <div><h4><Link to="/about/background">About</Link></h4></div>
+          </div>
         </div>
         <div className="footer-contact">
-          <div className="footer-mailer">
-            Receive updates on news, datasets, and features?
             <form action="//datawheel.us12.list-manage.com/subscribe/post?u=458059cf58ada1fd2b7de3e39&amp;id=9a3f9a9f32" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" className="validate" target="_blank" noValidate="">
               <div id="mc_embed_signup_scroll">
                 <div className="input-container">
+                  <input type="email" defaultValue="" name="EMAIL" className="email" id="mce-EMAIL" placeholder="RECEIVE UPDATES ON NEWS, DATASETS, AND FEATURES?" required="" />
                   <span className="bp3-icon bp3-icon-envelope"></span>
```

Comment:
Can we replace this blueprint icon with an SVG of Gaby's envelope exported from Figma? (and be careful about the left/right margins here, right now it's butting right up against the text and there should be some space there)